packages/node_modules/@ciscospark/react-component-activity-share-files/src/index.js

```diff
@@ -43,14 +47,16 @@ export function ActivityShareFiles(props) {
     isPending,
     onDownloadClick,
     share,
-    timestamp
+    timestamp,
+    cards,
+    renderAdaptiveCard
   } = props;
 
   const items = files.map((file) => {
     let objectUrl;
     let isFetching = true;
 
-    if (file.image) {
+    if (file.image && file.type !== 'cardReference') {
```

Comment:
Can we get this file type added as a constant?

Reply:
@adamweeks added requested changes and commit got broken down into different sections